kotlinKoans/src/main/java/com/epam/kotlinkoans/iii_conventions/n31Invoke.kt

```diff
@@ -1,19 +1,14 @@
 package iii_conventions
 
-import util.TODO
+class Invokable {
+    var numberOfInvocations: Int = 0
+        private set
 
+    operator fun invoke(): Invokable {
+        numberOfInvocations++
 
-class Invokable
-
-fun todoTask31(): Nothing = TODO(
-    """
-        Task 31.
-        Change the class 'Invokable' to count the number of invocations:
-        for 'invokable()()()()' it should be 4.
-    """,
-    references = { invokable: Invokable -> })
-
-fun task31(invokable: Invokable): Int {
-    todoTask31()
-//    return invokable()()()().getNumberOfInvocations()
```

Comment:
you should uncomment this line and don't change. Modification should be in Invokable class

Reply:
```suggestion
class Invokable {
    private var numberOfInvocations: Int = 0

    operator fun invoke(): Invokable {
        numberOfInvocations++

        return this
    }

    fun getNumberOfInvocations(): Int {
        return numberOfInvocations
    }
}

fun task31(invokable: Invokable): Int {
    return invokable()()()().getNumberOfInvocations()
}
```